src/Illuminate/Notifications/ChannelManager.php

```diff
@@ -74,7 +74,7 @@ public function sendNow($notifiables, $notification)
      * Queue the given notification instances.
      *
      * @param  mixed  $notification
-     * @param  array[\Illuminate\Notifcations\Channels\Notification]
+     * @param  array[\Illuminate\Notifcations\Channels\Notification]  $notification
```

Comment:
Why there are 2 `$notification` params in docblock? First function param named `$notifiable`.